Spec/Auth.swift

```diff
@@ -3619,5 +3619,331 @@ class Auth : QuickSpec {
                 }
             }
         }
+
+        describe("JWT and realtime") {
+            let channelName = "test_JWT"
+            let messageName = "message_JWT"
+            
+            context("client initialized with a JWT token in ClientOptions") {
+                let options = AblyTests.clientOptions()
+
+                context("with valid credentials") {
+                    options.token = getJWTToken()
+                    let client = AblyTests.newRealtime(options)
+                    defer { client.dispose(); client.close() }
+
+                    it("pulls stats successfully") {
+                        waitUntil(timeout: testTimeout) { done in
+                            client.stats{ stats, error in
+                                expect(error).to(beNil())
+                                done()
+                            }
+                        }
+                    }
+                }
+
+                context("with invalid credentials") {
+                    options.token = getJWTToken(invalid: true)
+                    options.autoConnect = false
+                    let client = AblyTests.newRealtime(options)
+                    defer { client.dispose(); client.close() }
+
+                    it("fails to connect with reason 'invalid signature'") {
+                        waitUntil(timeout: testTimeout) { done in
+                            client.connection.once(.failed) { stateChange in
+                                expect(stateChange!.reason!.code).to(equal(40144))
+                                expect(stateChange!.reason!.description).to(contain("invalid signature"))
+                                done()
+                            }
+                            client.connect()
+                        }
+                    }
+                }
+            }
+
+            // RSA8g RSA8c
+            context("when using auth_url") {
+                let options = AblyTests.clientOptions()
+                let keys = getKeys()
+                options.autoConnect = false
```

Comment:
Why are you setting `autoConnect` to false?  You did something similar in Ruby, is this needed in iOS for some reason?

Reply:
No needed. Done https://github.com/ably/ably-ios/pull/714/commits/5ec07607a40ef592fab887fb9b8fa5d02ec0be9a